lib/spire/version.rb

```diff
@@ -1,3 +1,3 @@
 module Spire
-  VERSION = "2.0.0"
+  VERSION = "2.0.1"
```

Comment:
@7chris71000 small comment. I think this should be '2.1.0' because we're introducing new features.

Reply:
@caseyli Version number has been addressed